src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -30,6 +30,7 @@
     public Map<String, String> emailNameTable = null;
     public Map<String, String> emailLastNameTable = null;
     public Map<String, String> emailTeamNameTable = null;
+    public Map<String, Set<String>> sectionTeamNameTable = null;
```

Comment:
It will be good if you add a comment somewhere explaining the difference between sectionTeamNameTable and rosterSectionTeamNameTable